### ISSUE TITLE
Incorporate tour into existing SPA Angular2+ code

### ISSUE THREAD

**Issue by michael-roschenwimmer** [isaacplmann/ngx-tour#46]
I'm hitting a block when trying to place the tour inside of an existing application (project) that is written in Angular 2+.

Would you please help guide me in layman's terms as to how to create a tour inside an application that is already working on its own? The current instructions might be more directed at mid-level or higher experience with Angular 2+... So maybe there are smaller steps that aren't listed as to how to accomplish this.

Please let me know if you need clarification on what I'm asking.

**Comment by isaacplmann** [comment]
What have you tried already?  What part isn't working?
